engines/native/sandbox.go

```diff
@@ -30,6 +30,25 @@ type sandbox struct {
 	wg         atomics.WaitGroup
 }
 
+// white list of environment variables that must
+// be passed to the child process
+var environmentWhitelist = []string{
+	"PATH",
+	"TMPDIR",
```

Comment:
PATH and TMPDIR makes sense to me to passthrough, the rest not much...

Setting a `HOME`, `EDITOR`, `LANG`, `LOGNAME`,..... variables as outlined in my command referrring to GNU su, also makes sense... probably with some of them being configurable.